dials.go

```diff
@@ -24,6 +24,13 @@ type Params struct {
 	//  - a Verify() method fails after re-stacking when a new version is
 	//    provided by a watching source
 	OnWatchedError WatchedErrorHandler
+
+	// SkipInitialVerification doesn't run the initial verification that would
+	// generally happen while calling `Config`.  This is useful when there are
```

Comment:
Let's call out the `Verify()` method (and the `VerifiedConfig interface) directly. This comment feels very indirect.
Also, I think it would make more sense to talk about watching sources here (with `Blank` being mentioned as a specific example).

Reply:
OK -- I've rewritten that in an attempt to be clearer.  Let me know what you think...